### ISSUE TITLE
Actions are not supported

### ISSUE THREAD

**Issue by boc-tothefuture** [boc-tothefuture/openhab-jruby#8]

**Comment by jimtng** [comment]
My experimental additions to OSGI seem to work, and I'm also experimenting with importing Actions, e.g. Audio, Voice, PersistenceExtensions, NotificationAction, etc. I'll need help in making it all nicely structured and I'll definitely need help in making the cucumber tests ;)

**Comment by boc-tothefuture** [comment]
audio and voice shouldn't need to use OSGI.

```
"scriptExtension", "automationManager", "ruleRegistry",
            "items", "voice", "rules", "things", "events", "itemRegistry", "ir", "actions", "se", "audio",
            "lifecycleTracker")
```

Those items are available as global variables.

Notifications I think does need to use OSGI

**Comment by jimtng** [comment]
Thanks! It should be noted in the README, along with examples

```
$voice.say("Hello World")
```

**Comment by boc-tothefuture** [comment]
I think we would want to hide the global variable and provide a wrapper around it.

**Comment by boc-tothefuture** [comment]
Resolved by #49
